### DISCUSSION TITLE
Espressopp weekly meeting (4.00 pm, 6. Okt)

### DISCUSSION THREAD

**Discussion by xzhh** [espressopp/espressopp#465] [Team Posts]
Unsure whether there will be full attendance, but feel free.
_____
Join Zoom Meeting
https://zoom.us/j/92605124006?pwd=c1dlR3JDUHlIYjJXeEVIcGlLaGc1QT09

Meeting-ID: 926 0512 4006
Kenncode: 120525

**Comment by AndreBrinkmann** [comment]
I am today travelling and can therefore unfortunately not attend. Nevertheless, I already discussed some next steps yesterday with Torsten.

 

Best 

 

André

 

Von: Zhen-Hao Xu ***@***.***>
Antworten an: espressopp/discussion ***@***.***>
Datum: Donnerstag, 6. Oktober 2022 um 10:02
An: espressopp/discussion ***@***.***>
Cc: Subscribed ***@***.***>
Betreff: [espressopp/discussion] Espressopp weekly meeting (4.00 pm, 6. Okt) (#11)

 

Unsure whether there will be full attendance, but feel free.

Join Zoom Meeting
https://zoom.us/j/92605124006?pwd=c1dlR3JDUHlIYjJXeEVIcGlLaGc1QT09

Meeting-ID: 926 0512 4006
Kenncode: 120525

**Comment by XzzX** [comment]
I have not heard anything from Luis so there is also no progress on my 
side for this week. Since its complicated for me to join today I will 
skip the meeting.

Best
Sebastian

On 06.10.22 10:03, AndreBrinkmann wrote:
> I am today travelling and can therefore unfortunately not attend. 
> Nevertheless, I already discussed some next steps yesterday with Torsten.
> 
> 
> 
> Best
> 
> 
> 
> André
> 
> 
> 
> Von: Zhen-Hao Xu ***@***.***>
> Antworten an: espressopp/discussion ***@***.***>
> Datum: Donnerstag, 6. Oktober 2022 um 10:02
> An: espressopp/discussion ***@***.***>
> Cc: Subscribed ***@***.***>
> Betreff: [espressopp/discussion] Espressopp weekly meeting (4.00 pm, 6. 
> Okt) (#11)
> 
> 
> 
> Unsure whether there will be full attendance, but feel free.
> 
> Join Zoom Meeting
> https://zoom.us/j/92605124006?pwd=c1dlR3JDUHlIYjJXeEVIcGlLaGc1QT09
> 
> Meeting-ID: 926 0512 4006
> Kenncode: 120525
> 
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you are subscribed to this thread.Message 
> ID: ***@***.***>
> 
> —
> Reply to this email directly, view it on GitHub 
> <https://github.com/orgs/espressopp/teams/discussion/discussions/11/comments/1>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AAU3OTCO7LRPSOD56GHFOW3WB2BVDANCNFSM6AAAAAAQ6KSBFQ>.
> You are receiving this because you are subscribed to this thread.Message 
> ID: ***@***.***>
> 

-- 
Dr. Sebastian Eibl
Max Planck Computing and Data Facility (MPCDF)
Gießenbachstr. 2
D-85748 Garching
Germany

**Comment by mrampp** [comment]
it seems there is not much to discuss from the MPCDF point of view, and thus
I would also skip the meeting, unless anybody has something else/urgent to
discuss with us?

best regards,
Markus

On Thu, Oct 06 2022, Sebastian Eibl wrote:
> I have not heard anything from Luis so there is also no progress on my 
> side for this week. Since its complicated for me to join today I will 
> skip the meeting.
> 
> Best
> Sebastian
> 
> On 06.10.22 10:03, AndreBrinkmann wrote:
> > I am today travelling and can therefore unfortunately not attend. 
> > Nevertheless, I already discussed some next steps yesterday with Torsten.
> > 
> > 
> > 
> > Best
> > 
> > 
> > 
> > André
> > 
> > 
> > 
> > Von: Zhen-Hao Xu ***@***.***>
> > Antworten an: espressopp/discussion ***@***.***>
> > Datum: Donnerstag, 6. Oktober 2022 um 10:02
> > An: espressopp/discussion ***@***.***>
> > Cc: Subscribed ***@***.***>
> > Betreff: [espressopp/discussion] Espressopp weekly meeting (4.00 pm, 6. 
> > Okt) (#11)
> > 
> > 
> > 
> > Unsure whether there will be full attendance, but feel free.
> > 
> > Join Zoom Meeting
> > https://zoom.us/j/92605124006?pwd=c1dlR3JDUHlIYjJXeEVIcGlLaGc1QT09
> > 
> > Meeting-ID: 926 0512 4006
> > Kenncode: 120525
> > 
> > —
> > Reply to this email directly, view it on GitHub, or unsubscribe.
> > You are receiving this because you are subscribed to this thread.Message 
> > ID: ***@***.***>
> > 
> > —
> > Reply to this email directly, view it on GitHub 
> > <https://github.com/orgs/espressopp/teams/discussion/discussions/11/comments/1>, 
> > or unsubscribe 
> > <https://github.com/notifications/unsubscribe-auth/AAU3OTCO7LRPSOD56GHFOW3WB2BVDANCNFSM6AAAAAAQ6KSBFQ>.
> > You are receiving this because you are subscribed to this thread.Message 
> > ID: ***@***.***>
> > 
> 
> -- 
> Dr. Sebastian Eibl
> Max Planck Computing and Data Facility (MPCDF)
> Gießenbachstr. 2
> D-85748 Garching
> Germany
> 
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/orgs/espressopp/teams/discussion/discussions/11/comments/2
> You are receiving this because you are subscribed to this thread.
> 
> Message ID: ***@***.***>

-- 
best regards / schoene Gruesse,
Markus Rampp

--  
---------------------------------------------------------
Dr. Markus Rampp
Max Planck Computing and Data Facility (MPCDF)
Giessenbachstr. 2, D-85748 Garching bei Muenchen, Germany
Phone: +49-89-3299-1324           FAX: +49-89-3299-1301
E-Mail: ***@***.*** WWW: www.mpcdf.mpg.de
---------------------------------------------------------
